cypress/support/iterator.js

```diff
@@ -0,0 +1,41 @@
+import config from './config/services';
+import alwaysTests from '../integration/pages';
+
+const serviceHasPageType = (service, pageType) =>
+  config[service].pageTypes[pageType].path !== undefined;
+
+const smokeTest = (service, pageType) =>
```

Comment:
Couldn't tell what this function did where it was being used so had to read here and I'm still not sure

Reply:
It decides what pages are run during smoke test and what isnt, not sure of a new name, any ideas?